src/applications/disability-benefits/2346/schemas/address-schema.js

```diff
@@ -241,9 +277,48 @@ export const addressUISchema = (
           },
         },
       },
+      province: {
+        'ui:title': 'State/Province/Region',
+        'ui:options': {
+          hideIf: (formData, index) => {
+            let countryNamePath = `${path}.country`;
+            if (typeof index === 'number') {
+              countryNamePath = insertArrayIndex(countryNamePath, index);
+            }
+            const livesOnMilitaryBase = get(livesOnMilitaryBasePath, formData);
+            if (isMilitaryBaseAddress && livesOnMilitaryBase) {
+              return true;
+            }
+            const countryName = get(countryNamePath, formData);
+            return countryName === USA.name || !countryName;
+          },
+        },
+        'ui:required': (formData, index) => {
+          if (
+            formData.selectedAddress === 'newAddress' &&
+            (formData.newAddress.country &&
+              formData.newAddress.country !== 'United States')
+          ) {
+            let countryNamePath = `${path}.country`;
+            if (typeof index === 'number') {
+              countryNamePath = insertArrayIndex(countryNamePath, index);
+            }
+            const livesOnMilitaryBase = get(livesOnMilitaryBasePath, formData);
+            if (isMilitaryBaseAddress && livesOnMilitaryBase) {
+              return true;
+            }
+            const countryName = get(countryNamePath, formData);
+            return countryName !== USA.name || !countryName;
+          }
+          return false;
+        },
+        'ui:errorMessages': {
+          required: 'Province is required',
+        },
+      },
       postalCode: {
         'ui:required': callback,
-        'ui:title': 'Postal Code',
+        'ui:title': 'Zip Code',
```

Comment:
Does DLC ship to addresses outside the U.S.? Maybe I'm not recalling correctly, or because I didn't look at the full uiSchema, but shouldn't this uiSchema have both a zipCode and a postalCode entry? Zip codes are only used in the U.S.

Reply:
Yes they ship internationally.  I believe it was a design decision to change the label from zip code to postal code no matter where we are shipping. I can follow up with them.

Reply:
Just followed up with them. We're going to show "Postal Code" no matter where they are.

Reply:
Now I'm extra confused... this change made it "Zip" code not "Postal"

Reply:
Nevermind, I see the code was updated, but not for this comment thread.